static/app/components/organizations/pageFilterBar.tsx

```diff
@@ -12,15 +13,15 @@ interface PageFilterBarProps extends React.HTMLAttributes<HTMLDivElement> {
   condensed?: boolean;
 }
 
-const PageFilterBar = styled(({children, condensed, ...props}: PageFilterBarProps) => {
+const PageFilterBar = styled(({children, ...props}: PageFilterBarProps) => {
   return (
-    <StyledPageFilterBar condensed={condensed} {...props}>
+    <StyledPageFilterBar listSize={Children.count(children)} {...props}>
```

Comment:
I think the only alternative to counting children would be to move away from compound components into an `items` prop that’s an array where we would get the fixed length (or make children be required to be an array, that would work too).

I checked all usages and there are no in-between wrappers around multiple elements, so this works right now.